packages/tslint.json

```diff
@@ -1,33 +1,51 @@
 {
+  "defaultSeverity": "warning",
   "rules": {
     "class-name": true,
-    "comment-format": [true, "check-space"],
+    "comment-format": [
+      true,
+      "check-space"
+    ],
     "curly": true,
     "forin": true,
-    "indent": [true, "spaces"],
-    "max-line-length": [true, 180],
+    "indent": [
+      true,
+      "spaces"
+    ],
+    "max-line-length": [
+      true,
+      180
+    ],
     "no-trailing-whitespace": true,
-    "no-unused-expression": true,
-    "no-use-before-declare": true,
```

Comment:
I've switched to tslint 5.x and these rules were broken for tslint 4.x. Now we have a lot of warnings for them and mostly invalid like the declaration of interfaces after usages, that is not broken. So I've disabled `no-use-before-declare` and moved `no-unused-expression` to dev.tslint.json.